src/main/java/Event.java

```diff
@@ -0,0 +1,31 @@
+public class Event extends Task{
+    protected String startOfEvent;
+    protected String endOfEvent;
+
+    public Event(String description, String startOfEvent, String endOfEvent) {
+        super(description);
+        this.startOfEvent = startOfEvent;
+        this.endOfEvent = endOfEvent;
+        this.taskType = "[E]";
+    }
+
+    public String getStartOfEvent() {
+        return startOfEvent;
+    }
+
+    public void setStartOfEvent(String startOfEvent) {
+        this.startOfEvent = startOfEvent;
+    }
+
+    public String getEndOfEvent() {
+        return endOfEvent;
+    }
+
+    public void setEndOfEvent(String endOfEvent) {
+        this.endOfEvent = endOfEvent;
```

Comment:
good job following the naming convention of the functions. Makes it easy to understand what they do.